### ISSUE TITLE
[Python] Unecessary PR for compatible version constraint

### ISSUE THREAD

**Issue by waghanza** [dependabot/feedback#809]
Hi,

We've introduce [compatible release](https://www.python.org/dev/peps/pep-0440/#compatible-release) in our codebase.

For example, in https://github.com/the-benchmarker/web-frameworks/blob/311315b8f4a73e822d0530bccbac19f2ec09dfd0/python/django/requirements.txt#L1, we've specified `django~= 3.0.0` which means that we want all versions between **3.0** and **3.1**.

As a results, we might not want any change until any `django` release in serie **3.1** is released.

The `PR` https://github.com/the-benchmarker/web-frameworks/pull/2075 created by _dependabot_ in then not necessary.

Regards,

**Comment by feelepxyz** [comment]
@waghanza yes we'll need to extend support to configure `version_requirement_updates: increase_versions_if_necessary ` for Python. This is currently only possible for php, ruby and js: https://dependabot.com/docs/config-file/#version_requirement_updates

Adding this is as an enhancement as it's currently expected behaviour.

**Comment by waghanza** [comment]
+ `django~=3.0.0`
+ `django>=3.0.0,<3.1.0`

is equivalent according to [pep 440](https://www.python.org/dev/peps/pep-0440/#compatible-release)

I do not know how version constraints are parse, but it's seems not to respect this **PEP**

the first solution lead to unnecessary `PR`, but the second is OK

I will change my codebase then https://github.com/the-benchmarker/web-frameworks/issues/2123
